.tests/compatibility/config.yaml

```diff
@@ -0,0 +1,31 @@
+samples: "samples.tsv"
+units: "units.tsv"
+resources: "resources.yaml"
+
+trimmer_software: "fastp_pe"
+
+default_container: "docker://hydragenetics/common:0.1.2"
```

Comment:
```suggestion
default_container: "docker://hydragenetics/common:0.1.3"
```

Reply:
I missed this, sorry. Just added a new PR 😬